bibliography.bib

```diff
@@ -7,6 +7,26 @@ @inproceedings{gullasch2011cache
   organization={IEEE}
 }
 
+@incollection{ristenpart2009cloud,
+  title={Hey, you, get off of my cloud: exploring information leakage in third-party compute clouds},
+  author={Ristenpart, Thomas and Tromer, Eran and Shacham, Hovav and Savage, Stefan},
+  booktitle={CCS '09 Proceedings of the 16th ACM conference on Computer and communications security},
+  pages={199--212},
+  year={2009},
+  publisher={ACM},
+  annote={
+    \setlength{\parskip}{1.5ex}
```

Comment:
I'm not sure that this looks better. If you think it does, we should find a way of putting it in our preamble once instead of starting every annotation with it.